support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -173,93 +161,8 @@ class DigitalPackEmailFields(
     EmailFields(attributePairs, user, dataExtensionName, userAttributes)
```

Comment:
I don't think `userAttributes` is needed any more
I think we also did work in membership-workflow to push unmanaged attributes to braze
we could take a view on whether it's worth keeping these things?  I remember this route to getting things into emails was frowned upon heavily at the time though.

Reply:
I'm not going to change that in this PR because I don't really understand what it's doing. It looks like all products do pass something into the userAttributes, mostly just a string of the product name